test/sinon-test.js

```diff
@@ -10,34 +10,61 @@ if (!hasPromise) {
 
 describe("sinon module", function () {
     var sinon,
+        fakeSandbox,
         fakeNise;
 
     beforeEach(function () {
 
         fakeNise = {
-            fakeServer: "47c86a4c-6b48-4748-bb8c-d853f999720c",
-            fakeServerWithClock: "e69974f8-4568-48d1-a5e9-2b511a59c14b",
+            fakeServer: {
+                create: "47c86a4c-6b48-4748-bb8c-d853f999720c"
+            },
+            fakeServerWithClock: {
+                create: "e69974f8-4568-48d1-a5e9-2b511a59c14b"
+            },
             fakeXhr: {
                 xhr: "958e8996-0cc3-4136-8a0e-6a120f5311bc",
                 FakeXMLHttpRequest: "6adbf569-f6d7-4b86-be22-38340ae0f8c8",
                 useFakeXMLHttpRequest: "ba8bd609-c921-4a62-a1b9-49336bd426a4"
             }
         };
+        fakeSandbox = {
+            create: "dc61d622-407f-46eb-af24-7a83bb30b8bf"
+        };
         sinon = proxyquire("../lib/sinon", {
-            nise: fakeNise
+            nise: fakeNise,
+            "./sinon/sandbox": fakeSandbox
```

Comment:
I have to go through proxyquire to get the correct reference to the sandbox instance. I tried using a reference to the sandbox module directly, which worked on node but it failed in browsers.